pss/pss.go

```diff
@@ -52,11 +59,41 @@ func (p *Pss) Send(ctx context.Context, targets [][]byte, topic trojan.Topic, pa
 		return nil, err
 	}
 
+	tag := chunk.NewTag(1, "pss-chunks-tag", 0, false)
+	tc.WithTagID(tag.Uid)
+
 	// SAVE trojanChunk to localstore, if it exists do nothing as it's already peristed
-	// TODO: for second phase, use tags --> listen for response of recipient, recipient offline
 	if _, err = p.localStore.Put(ctx, chunk.ModePutUpload, tc); err != nil {
 		return nil, err
 	}
+	tag.Inc(chunk.StateStored)
```

Comment:
let's get @janos and @zelig's input here: i think we are both wondering why we need to manually call `Inc` at this point.

i would expect another part of the system to actually change these tags, and us just to monitor this as it happens.

i suppose we are missing something. i'll try to investigate the `tag` code in the meantime

Reply:
That was my first assumption it was called automatically from other places, but if you look at `hasherStore` that is used by `filestore` you can find it calling it correctly.
Since we are using the underlying localstore it's not being updated, maybe we need to use a higher abstraction and call that instead.

Reply:
- here it is the number of chunks that are STORED is incremented
- those other compoments where SENT and SYNCED are incremented are here: 
- https://github.com/ethersphere/swarm/blob/master/pushsync/pusher.go#L351
- https://github.com/ethersphere/swarm/blob/master/storage/localstore/mode_set.go#L272

Reply:
removed, I will add it as a note in the PR, just in case we change it later on.